src/main/java/edu/ucsb/cs156/example/controllers/PersonalSchedulesController.java

```diff
@@ -0,0 +1,101 @@
+package edu.ucsb.cs156.example.controllers;
+
+import edu.ucsb.cs156.example.entities.PersonalSchedule;
+import edu.ucsb.cs156.example.entities.User;
+import edu.ucsb.cs156.example.errors.EntityNotFoundException;
+import edu.ucsb.cs156.example.models.CurrentUser;
+import edu.ucsb.cs156.example.repositories.PersonalScheduleRepository;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import lombok.extern.slf4j.Slf4j;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+@Api(description = "API to handle CRUD operations for Personal Schedules database")
+@RequestMapping("/api/personalschedules")
+@RestController
+@Slf4j
+public class PersonalSchedulesController extends ApiController {
+
+    @Autowired
+    PersonalScheduleRepository PersonalScheduleRepository;
```

Comment:
Can you rename this field (any other variable names in this file) to use `lowerCamelCase`? It can get a little confusing to read the code when you break from standard Java naming conventions.

Reply:
Fixed it to lowerCamelCase.